polyfill/spatial-navigation-polyfill.js

```diff
@@ -1193,26 +1193,24 @@
       /* falls through */
     case 'right' :
       // If two elements are aligned, add align bias
-      if (isAligned(rect1, rect2, dir)) {
-        alignBias = intersectionRect.height / rect1.height;
-        if (alignBias > 1) align_bias = 1;
-      }
-      else  // else, add orthogonal bias
+      // else, add orthogonal bias
+      if (isAligned(rect1, rect2, dir))
+        alignBias = (alignBias > 1) ? 1 : (intersectionRect.width / rect1.width);
```

Comment:
"width" -> "height"

and, how about
  > alignBias =  Math.min(intersectionRect.height/rect1.height , 1);

because alignBias's initial value is 0, the true statement will not be invoked.

Reply:
Yes, it's more precise. Thanks, I'll fix it.